tests/test_kubernetes.py

```diff
@@ -504,7 +504,7 @@ def test__build_cache(self, mock_read_chunked):
             {'type': 'DELETED', 'object': {'metadata': {
                 'name': self.k.config_path, 'resourceVersion': '3'}}}
         ) + '\n' + json.dumps(
-            {'type': 'MDIFIED', 'object': {'metadata': {'name': self.k.config_path}}}
+            {'type': 'MODIFIED', 'object': {'metadata': {'name': self.k.config_path}}}
```

Comment:
This one is intentional.
```suggestion
            {'type': 'MDIFIED', 'object': {'metadata': {'name': self.k.config_path}}}
```